node/src/components/binary_port.rs

```diff
@@ -148,7 +183,6 @@ where
         + From<ChainspecRawBytesRequest>
         + Send,
 {
-    let protocol_version = effect_builder.get_protocol_version().await;
```

Comment:
I think we don't need to call this since we can store the protocol version directly from chainspec in `new`.

Reply:
Agreed